app/models/site_config.rb

```diff
@@ -66,6 +66,7 @@ class SiteConfig < RailsSettings::Base
   # Emails
   field :email_addresses, type: :hash, default: {
     default: ApplicationConfig["DEFAULT_EMAIL"],
+    contact: ApplicationConfig["DEFAULT_EMAIL"],
```

Comment:
Love that I didn't have to do anything for this field to appear in the SiteConfig page ❤️